tests/integration/column_mapping/test_pipeline_column_mapping.py

```diff
@@ -0,0 +1,237 @@
+import shutil
+from pathlib import Path
+
+from digital_land.log import ColumnFieldLog
+from digital_land.pipeline import Pipeline
+from digital_land.specification import Specification
+
+from tests.integration.column_mapping.helper.funcs import prepare_data, process_pipeline
+
+import pytest
+
+
+@pytest.fixture
+def env_vars(tmp_path, request):
+    root_dir = request.config.rootdir
+    test_root = Path(root_dir).resolve()
+
+    target_endpoint = "d779ad1c91c5a46e2d4ace4d5446d7d7f81df1ed058f882121070574697a5412"
+    dummy_endpoint = "abcde12345fghij67890abcde12345fghij67890abcde12345fghij67890abcd"
+    endpoints = [dummy_endpoint, target_endpoint, dummy_endpoint[::-1]]
+
+    return {
+        "endpoints": endpoints,
+        "resource": "5158d13bfc6f0723b1fb07c975701a906e83a1ead4aee598ee34e241c79a5f3d",
+        "dataset_name": "article-4-direction-area",
+        "column_all_csv": f"{tmp_path}/data/column_all.csv",
+        "column_res_csv": f"{tmp_path}/data/column_res.csv",
+        "column_eps_csv": f"{tmp_path}/data/column_eps.csv",
+        "column_csv": f"{tmp_path}/pipeline/column.csv",
+        "pipeline_dir": f"{tmp_path}/pipeline/",
+        "specification_dir": f"{test_root}/tests/integration/column_mapping/specification/",
+        "output_dir": f"{tmp_path}/output/",
+    }
+
+
+@pytest.fixture(autouse=True)
+def before_and_after_tests(tmp_path):
+    """Fixture to execute asserts before and after a test is run"""
+    # Setup:
+    data_dir = tmp_path / "data"
+    data_dir.mkdir()
+
+    pipeline_dir = tmp_path / "pipeline"
+    pipeline_dir.mkdir()
+
+    output_dir = tmp_path / "output"
+    output_dir.mkdir()
+
+    yield
+
+    # Teardown:
+    pass
+
+
+def test_required_external_asset_creation(env_vars):
```

Comment:
We don't need to test that the specification is being read in properly here. That would be done in the specification testing.

Reply:
Accepted.
Happy to remove the test if it's redundant.